igb_client/parse_custom.py

```diff
@@ -0,0 +1,45 @@
+from typing import Any, Dict, List, Set
+from collections import defaultdict
+
+
+def params_parser(payload: Dict) -> Dict[str, List[Any]]:
+    """
+    Converts from:
+    {
+        "name": "\\IGB\\Property\\Custom\\Doelsite\\Vdab\\Sjabloon",
+        "params": [
+            {"param": "credentials"},
```

Comment:
is it ok to lose the `credentials` param after parsing?

Reply:
Updated the description and the method names so that it's more clear what it does.

Check this unit test (put in PKB where we have pipelines) https://github.com/vonq/pkb/pull/702/files#diff-4dca771315e860a38d9375c36feac13f3d4419de9a9234bae5dad14eefb7c6c7R2

Reply:
The idea is that until now, "term" was used to take value from the user input.
With vdab, we use the vacancy title or the value from other facets.